mono/utils/mono-threads-coop.c

```diff
@@ -703,7 +702,7 @@ mono_threads_suspend_validate_policy (MonoThreadsSuspendPolicy policy)
 void
 mono_threads_suspend_override_policy (MonoThreadsSuspendPolicy new_policy)
 {
-	threads_suspend_policy = (char)mono_threads_suspend_validate_policy (new_policy);
+	mono_threads_suspend_policy_hidden_dont_modify = (char)mono_threads_suspend_validate_policy (new_policy);
```

Comment:
Avoid contractions as too informal? `do_not` instead of `dont`.

Reply:
There are other names for this in practise also, like "mostly readonly". abbreviated imho obscurely as "mr".
Often times such data is actually made read only after initialization with VirtualProtect. Though there are considerations for that, i.e. isolating it to a page with also mostly readonly.

Reply:
You can also play useful games like declaring it as const but implementating it as non const.
With externC. i.e. to enforce not writing to it, except where allowed.